ci/teamcity/comment_on_pr.py

```diff
@@ -23,22 +23,34 @@
 
 pytest_outputs = ["ray_tests.log", "dask_tests.log", "python_tests.log"]
 
-full_comment = header
+full_comment = ""
+# Do not include coverage info in PR comment
+split_by = "----------- coverage: platform linux, python 3.7.5-final-0 -----------"
 
 for out in pytest_outputs:
 
     full_comment += "<details><summary>{} Tests</summary>\n".format(
         out.split("_")[0].title()
     )
     full_comment += "\n\n```\n"
-    full_comment += open(out, "r").read()
+    content = open(out, "r").read()
+    full_comment += content.split(split_by)[0]
     full_comment += "\n```\n\n</details>\n"
 
+if "FAILURES" not in full_comment:
+    header += "<b>Tests PASSed</b>\n\n"
+else:
+    header += "<b>Tests FAILed</b>\n\n"
+
+full_comment = header + full_comment
+
 token = os.environ["GITHUB_TOKEN"]
 g = Github(token)
 repo = g.get_repo("modin-project/modin")
 
 pr = repo.get_pull(pr_id)
+if len(full_comment) > 65000:
+    full_comment = full_comment[:65000] + "\n\nRemaining output truncated\n```\n"
```

Comment:
I thought that it is end of the log that contains more valuable information like list of failed tests or exception stack traces.